external/derby/README.md

```diff
@@ -0,0 +1,14 @@
+## Running External tests locally
```

Comment:
Before this section add:

# External Derby Tests

Derby tests are a part of the external third-party application tests that help verify that the Adoptium binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/[Issue #172](https://github.com/AdoptOpenJDK/openjdk-tests/issues/172) lists the applications that we have initially targeted to best exercise the Adoptium binaries. For each application, we choose to run a selection of their functional tests. Derby test material is pulled from the [derby](https://github.com/apache/derby.git) repository.


Then also change the heading "Running External tests locally" to be "Running Derby tests locally"